AdminUI/Dockerfile.debug

```diff
@@ -0,0 +1,13 @@
+# Build stage
+FROM node:16-alpine as build-stage
```

Comment:
node 18 is the current LTS version. Why don't you use that one?

Reply:
Remnants of debugging, I'll fix it.